bundles/core/org.eclipse.smarthome.core/src/main/java/org/eclipse/smarthome/core/scheduler/ThreadPoolManager.java

```diff
@@ -5,16 +5,25 @@
  * which accompanies this distribution, and is available at
  * http://www.eclipse.org/legal/epl-v10.html
  */
-package org.eclipse.smarthome.core.common;
+package org.eclipse.smarthome.core.scheduler;
```

Comment:
This class should imho stay where it was, so that we keep backward compatibility.
So we could either 
a) have a dependency from core.common to core.scheduler by this
b) have getExpressionScheduledPool() return a ThreadPoolExecutor that needs to be casted by the caller
c) Introduce a core.scheduler.ExpressionThreadPoolManager class that extends the normal one.

No solution looks ideal to me, so discussion is opened :-)